preprocessing/berlin/compile-berlin-geojson.js

```diff
@@ -169,6 +175,22 @@ function composeOpeningHours(days, hours) {
 	throw "Day can most likely not be processed automatically.";
 }
 
+/*
+ * Returns a sanitized HTTPS URL or an empty string.
+ * HTTPS is always used, because that's want everyone should use.
+ */
+function getSanitizedUrl(url) {
+	url = getSanitizeString(url);
+	if (!url) {
+		return url;
+	}
+	url = url.replace("http://", "https://");
```

Comment:
I don't think we can assume that HTTPS is supported everywhere! Some simple websites from a village somewhere might not yet be there!

I'm against replacing protocols here. We could allow only HTTPS but this should be handled during linting or tests and not at runtime!

I don't think we need the `getSanitizedUrl` at all here, because there is still the review process in place.

Reply:
This preprocessing script is Berlin specific. None of the other cities automated the market data generation to my knowledge.